### PR TITLE
Compute tables of available converters per opset

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: skl2onnx/validate.py [hunk 1]
```diff
@@ -0,0 +1,827 @@
+# -------------------------------------------------------------------------
```
Comment (prabhat00155) [inline skl2onnx/validate.py]: Could you increase test coverage on this file? Currently 74 lines in this file haven't been tested by unit tests.
skl2onnx/validate.py                                         399     74    81%

File: tests/test_validate.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        assert len(res) > 0
+        assert len(res[0]) == 3
+
+    @unittest.skipIf(StrictVersion(onnx.__version__) < StrictVersion("1.4.0"),
+                     reason="OnnxOperator not working")
+    @unittest.skipIf(("dev" not in sklearn.__version__ and
+                      StrictVersion(sklearn.__version__) <
+                      StrictVersion("0.21")),
+                     reason="needed only once")
+    @unittest.skipIf(not has_openpyxl(), reason="missing dependency")
```
Comment (wenbingl) [inline tests/test_validate.py]: Given you already put openyxl in the requirements-dev.txt, please remove this skip line.
Too many skip lines will make other fully be lost about how to run the full test.

File: skl2onnx/validate.py [hunk 3]
```diff
@@ -0,0 +1,827 @@
...
+
+def get_opset_number_from_onnx():
+    """
+    Retuns the current *onnx* opset
+    based on the installed version of *onnx*.
+    """
+    return onnx.defs.onnx_opset_version()
+
+
+def sklearn_operators(subfolder=None):
```
Comment (adrinjalali) [inline skl2onnx/validate.py]: I guess you could use `sklearn.utils.testing.all_estimators`

File: tests/test_validate.py (hunk 2 above)
Comment (xadupre) [reply tests/test_validate.py]: Wrong copy paste, it checks pydot, py dot is part of the requirements, but it still needs graphviz which not necessarily installed.

File: docs/exts/sphinx_skl2onnx_extension.py [hunk 4]
```diff
@@ -253,6 +220,199 @@ def run(self):
...
+            else:
+                raise ValueError(
+                    "align should be 'l', 'r', 'c' not '{0}'".format(align))
+        else:
+            return s
+
+    def complete(cool):
+        if list_table:
+            i, s = cool
+            if s is None:
```
[the remark this answers is not in the document]
Comment (xadupre) [reply docs/exts/sphinx_skl2onnx_extension.py]: upper and lower part of the test looks the same, i prefer to keep it that way.

File: skl2onnx/validate.py (hunk 3 above)
Comment (xadupre) [reply skl2onnx/validate.py]: Fixed.

**Pull request closed without merging**